File: base16/crosh/colors/base16-sagelight.js
// Base16 Sagelight
// Scheme: Carter Veldhuizen

var color_scheme = {
        'base00': '#f8f8f8',
        'base01': '#e8e8e8',
        'base02': '#d8d8d8',
        'base03': '#b8b8b8',
        'base04': '#585858',
        'base05': '#383838',
        'base06': '#282828',
        'base07': '#181818',
        'base08': '#fa8480',
        'base09': '#ffaa61',
        'base0A': '#ffdc61',
        'base0B': '#a0d2c8',
        'base0C': '#a2d6f5',
        'base0D': '#a0a7d2',
        'base0E': '#c8a0d2',
        'base0F': '#d2b2a0',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(56, 56, 56, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);

File: base16/vis/themes/base16-sagelight.lua
-- base16-vis (https://github.com/pshevtsov/base16-vis)
-- by Petr Shevtsov
-- Sagelight scheme by Carter Veldhuizen

local lexers = vis.lexers

local colors = {
	['base00'] = '#f8f8f8',
	['base01'] = '#e8e8e8',
	['base02'] = '#d8d8d8',
	['base03'] = '#b8b8b8',
	['base04'] = '#585858',
	['base05'] = '#383838',
	['base06'] = '#282828',
	['base07'] = '#181818',
	['base08'] = '#fa8480',
	['base09'] = '#ffaa61',
	['base0A'] = '#ffdc61',
	['base0B'] = '#a0d2c8',
	['base0C'] = '#a2d6f5',
	['base0D'] = '#a0a7d2',
	['base0E'] = '#c8a0d2',
	['base0F'] = '#d2b2a0',
}

lexers.colors = colors

local fg = ',fore:'..colors.base05..','
local bg = ',back:'..colors.base00..','

lexers.STYLE_DEFAULT = bg..fg
lexers.STYLE_NOTHING = bg
lexers.STYLE_CLASS = 'fore:'..colors.base0A
lexers.STYLE_COMMENT = 'fore:'..colors.base03..',italics'
lexers.STYLE_CONSTANT = 'fore:'..colors.base09
lexers.STYLE_DEFINITION = 'fore:'..colors.base0E
lexers.STYLE_ERROR = 'fore:'..colors.base08..',italics'
lexers.STYLE_FUNCTION = 'fore:'..colors.base0D
lexers.STYLE_KEYWORD = 'fore:'..colors.base0E
lexers.STYLE_LABEL = 'fore:'..colors.base0A
lexers.STYLE_NUMBER = 'fore:'..colors.base09
lexers.STYLE_OPERATOR = 'fore:'..colors.base05
lexers.STYLE_REGEX = 'fore:'..colors.base0C
lexers.STYLE_STRING = 'fore:'..colors.base0B
lexers.STYLE_PREPROCESSOR = 'fore:'..colors.base0A
lexers.STYLE_TAG = 'fore:'..colors.base0A
lexers.STYLE_TYPE = 'fore:'..colors.base0A
lexers.STYLE_VARIABLE = 'fore:'..colors.base0D
lexers.STYLE_WHITESPACE = 'fore:'..colors.base02
lexers.STYLE_EMBEDDED = 'fore:'..colors.base0F
lexers.STYLE_IDENTIFIER = 'fore:'..colors.base08

lexers.STYLE_LINENUMBER = 'fore:'..colors.base02..',back:'..colors.base00
lexers.STYLE_CURSOR = 'fore:'..colors.base00..',back:'..colors.base05
lexers.STYLE_CURSOR_PRIMARY = 'fore:'..colors.base00..',back:'..colors.base05
lexers.STYLE_CURSOR_LINE = 'back:'..colors.base01
lexers.STYLE_COLOR_COLUMN = 'back:'..colors.base01
lexers.STYLE_SELECTION = 'back:'..colors.base02
lexers.STYLE_STATUS = 'fore:'..colors.base04..',back:'..colors.base01
lexers.STYLE_STATUS_FOCUSED = 'fore:'..colors.base09..',back:'..colors.base01
lexers.STYLE_SEPARATOR = lexers.STYLE_DEFAULT
lexers.STYLE_INFO = 'fore:default,back:default,bold'
lexers.STYLE_EOF = ''
